src/main.py

```diff
@@ -606,8 +605,9 @@ def cleanup(signal, frame):   # pylint: disable=redefined-outer-name,unused-argu
 	pHandler.setup()
 	platform.setup()
 
+	logger.info("Checking Internet Connection ...")
 	while not internet_on():
```

Comment:
So if we encounter unresolvable condition, like `ProxyError`, we will try again? I think that we should be specific on which errors we should retry and others, where retrying is pointless.